arctic/serialization/numpy_records.py

```diff
@@ -29,12 +29,15 @@ def set_fast_check_df_serializable(config):
 
 def _to_primitive(arr, string_max_len=None):
     if arr.dtype.hasobject:
-        if len(arr) > 0:
-            if isinstance(arr[0], Timestamp):
+        if len(arr) > 0 and isinstance(arr[0], Timestamp):
                 return np.array([t.value for t in arr], dtype=DTN64_DTYPE)
         if string_max_len:
```

Comment:
nit: either add a blank line or make this an elif.

Reply:
changed

Reply:
Still indented by an extra level

Reply:
thanks @lwfitzgerald  , fixed now